### PR TITLE
Finalize Preprocessing

### PR DESCRIPTION
**Pull request by jankokla**
This PR will merge the work of Paul and Valentin. I reordered some things and moved most of the functionality to the notebook for better visibility (for graders). Please take a look and let me know if it looks ok!

### CODE REVIEW

File: notebooks/classification.ipynb
Comment (paulbogdy) [inline notebooks/classification.ipynb:1]: Is this notebook really necessary ? It seems to have some errors

File: notebooks/preprocess.ipynb
Comment (paulbogdy) [inline notebooks/preprocess.ipynb:1]: Looks really great, thanks for the reorganization.

**Review by paulbogdy — APPROVED** [review]
Looks really good, great job and thanks.

File: notebooks/classification.ipynb
Comment (jankokla) [reply notebooks/classification.ipynb:1]: Good point, they need updating. I'll take a look tomorrow.

**Pull request merged**